packages/tradershub/src/features/cfd/flows/MT5/AvailableMT5AccountsList/AvailableMT5AccountsList.tsx

```diff
@@ -15,7 +22,11 @@ const AvailableMT5AccountsList = ({ account }: { account: THooks.MT5AccountsList
                 <Button
                     className='rounded-200'
                     colorStyle='coral'
-                    variant='primary' /* onClick show MT5PasswordModal : JurisdictionModal */
+                    onClick={() => {
+                        setCfdState('marketType', account.market_type);
+                        !activeAccount?.is_virtual && show(<JurisdictionModal />); /* show MT5PasswordModal for demo */
```

Comment:
We also should be handling if its !svg

Reply:
I don't get it, can you elaborate ?

Reply:
Because in svg jurisdiction we dont show jurisdiction, straight away the password modal 😸